packages/rest/README.md

```diff
@@ -52,16 +52,44 @@ Application options.
 const app = new RestApplication({
   rest: {
     port: 3001,
+
+    // Template of the OpenAPI spec, which will be filled with `paths` from
```

Comment:
I am proposing to move this new documentation to https://github.com/strongloop/loopback-next/blob/master/docs/site/Server.md and modify this README with a link pointing to that doc page.

For example, configuration of HTTPS is already described in `Server.md` only, see [here](https://github.com/strongloop/loopback-next/blob/ac011f80b229b5dd5c69851ab14b39705b852629/docs/site/Server.md#enable-https).